geth/jail/handlers.go

```diff
@@ -66,23 +66,54 @@ func registerHandlers(jail *Jail, cell common.JailCell, chatID string) error {
 	return nil
 }
 
+// makeAsyncSendHandler returns jeth.sendAsync() handler.
+func makeAsyncSendHandler(jail *Jail, cellInt common.JailCell) func(call otto.FunctionCall) otto.Value {
+	// FIXME(tiabc): Get rid of this.
+	cell := cellInt.(*Cell)
+	return func(call otto.FunctionCall) otto.Value {
+		go func() {
+			response := jail.Send(call, cell.VM)
```

Comment:
I wonder if we shouldn't call this outside of a goroutine and return the result... It usually should be `undefiend` but who knows.

Reply:
No, we shouldn't. This invocation is actually the main point of this PR which introduced all the other changes.

Reply:
Oh ok. I get it now after giving it a little bit more thinking :)